README.md

```diff
@@ -16,15 +16,18 @@ Installation
 
 Once MESA has been installed:
 * copy the ``run_star_extras.f`` and ``mesa_hdf5_*.inc`` files located in the ``src`` folder
-into the ``src`` folder in your work directory
+into the ``src`` folder in your work directory.
+* copy the ``hdf5_profile_columns.list`` file located in the ``test`` folder into the MESA working directory and [update](https://github.com/NuGrid/mesa_h5#profiles-in-the-hdf5-file) the ``profile_columns.list`` MESA file.
 * clean, compile, and enjoy!
 
 Profiles in the HDF5 file
 -------------------------
 
-The user must specify which MESA profiles should be included in the HDF5. This is done in a file (default name ``hdf5_profile_columns.list`` similar to the ``profile_columns.list`` MESA file.
+The user must specify which MESA profiles should be included in the HDF5. This is done in the file ``hdf5_profile_columns.list`` (default name) similar to the ``profile_columns.list`` MESA file.
 
-Note that the profiles in the HDF5 file **MUST** be a subset of the profiles written out in the MESA ASCII files. So whatever contains ``hdf5_profile_columns.list`` should also be in ``profile_columns.list``.
+Note that the profiles in the HDF5 file **MUST** be a subset of the profiles written out in the MESA ASCII files. So whatever contains ``hdf5_profile_columns.list`` should also be either in ``profile_columns.list`` or is calculated in the extra routines. The ``profile_columns.list`` needs to include ``dm``, ``radius``, ``rmid``, ``temperature``, ``log_D_mix``, ``mass`` and ``rho``.
+
+Also the MESA ``profile_columns.list`` **MUST** contain ``time_step``.
```

Comment:
Are those requirements for any MESA output or are they specific to NuGrid data? If the latter, if should be clarified.

Reply:
The first list (from "dm" to "rho") needs to be included in the MESA output because they are in the default "hdf5_profile_columns.list" (which is specific to NuGrid data I guess). I wrote this paragraph because there are some profiles in the "hdf5_profile_columns.list" that are calculated by the mesa_h5 routines and it might be helpful to specify for new users.
So I agree with you that we might clarify this in the documentation.

The second part ("time_step") needs to be included for any MESA output. This parameter is not listed in the "hdf5_profile_columns.list" and the run will create the HDF5 files. However, when I used them to post-process with mppnp there was an error (saying that I did not include "detlat" aka "time_step" in MESA).

```suggestion
Note that the profiles in the HDF5 file **MUST** be a subset of the profiles written out in the MESA ASCII files. So whatever contains ``hdf5_profile_columns.list`` should also be either in ``profile_columns.list`` or is calculated in the extra routines. 
For example, when the default ``hdf5_profile_columns.list`` is used the ``profile_columns.list`` needs to include ``dm``, ``radius``, ``rmid``, ``temperature``, ``log_D_mix``, ``mass`` and ``rho``. The other profiles (``delta_mass`` and ``dcoeff``) are calculated by the subroutines provided in this package.

Also the MESA ``profile_columns.list`` **MUST** contain ``time_step`` if the HDF5 output will be used for [post-processing](https://github.com/NuGrid/NuPPN).
```

I hope this clarifies is, let me know if not.

Reply:
All good, thanks! :)